practice/01-js-intro-and-jquery/script.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready(function(){
+
+
+	$('section').fadeIn(3000);
+
+	$('.alias').focus();
+
+	$("button").click( function() {
+		$.ajax('http://bootcamp.aws.af.cm/welcome/nicolas', {
+     	 success: function(result) {
+        $('section').html(result.response);
+      },
+      	 error: function(){
+
+      	 	$('section').css('font-color','red');
+
+      	 },
+         complete: function(){
+
+          $('section').highlight('nicolas');
+
+
+
+
+           }});
+      });
+
+       $.ajax('https://api.spotify.com/v1/search',{data: "q=Rolling Stones&type=album"},
```

Comment:
This had to ignore ahaha, I was trying to do the next exercise when I did the commit